.env.example

```diff
@@ -9,3 +9,10 @@ OPENAI_API_KEY=""
 # Optional but required if send_email action is used
 RESEND_API_KEY=""
 RESEND_EMAIL=""
+
+# Posthog
+ADMYRAL_POSTHOG_API_KEY="phc_RIpkRea4KLW6EONEDCSZVR1Td4YzeHf4ziUsGzmPnjD"
+ADMYRAL_POSTHOG_HOST="https://eu.i.posthog.com"
+
+# used for development to deactivate posthog logging
+ADMYRAL_ENV="dev"
```

Comment:
Just thought about it again, it would be better to have an additional environment variable for deactivating posthog (I would call it `ADMYRAL_DISABLE_TELEMETRY`). The reason is that the CLI command won't work for docker-compose hence we must have a flag for disabling it. Right now it doesn't matter because it is only used in the CLI but we should still set it up correctly.